fuel/app/config/config.php

```diff
@@ -196,18 +196,20 @@
 	/**
 	 * Cookie settings
 	 */
-	// 'cookie' => array(
+	'cookie' => array(
 		// Number of seconds before the cookie expires
 		// 'expiration'  => 0,
 		// Restrict the path that the cookie is available to
 		// 'path'        => '/',
 		// Restrict the domain that the cookie is available to
 		// 'domain'      => null,
 		// Only transmit cookies over secure connections
-		// 'secure'      => false,
+		'secure'      => $_SERVER['HTTPS'] ?? false,
```

Comment:
Unfortunately I don't think relying on `$_SERVER['HTTPS']` is going to work for us, since this relies on the webserver running from the same machine as the PHP application. With the docker setup we have now, the webserver is running in its own container and forwarding requests to the PHP container through docker networking. The next best thing might be yet another env variable, but realistically I'm not sure there would ever be a context where someone wants to run the server application without https enabled. Perhaps setting the default to `true` is okay here, alongside an optional env variable override.